Aardvark/ARKEmailBugReporter.m

```diff
@@ -37,6 +37,82 @@ @interface ARKInvisibleView : UIView
 @end
 
 
+@interface CALayer (HierarchyDescription)
+
+- (NSString *)_recursiveLayerHierarchyDescriptionWithIndentationLevel:(int)indentationLevel;
```

Comment:
we should prefix the names of extensions with `ARK_` or `_ARK_` to prevent potential collisions with Apple's code.

Reply:
+1